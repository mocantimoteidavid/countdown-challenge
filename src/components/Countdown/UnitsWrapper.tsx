import styled from "styled-components"

const UnitsWrapper = styled.div`
  display: flex;
  flex-wrap: nowrap;
  align-items: center;
  justify-content: left;

  & > div {
    margin-right: 6px;
  }
  & > div:last-child {
    margin-right: 0;
  }
`

export default UnitsWrapper
