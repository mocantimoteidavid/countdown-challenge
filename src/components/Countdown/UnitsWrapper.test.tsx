import { render } from "@testing-library/react"
import UnitsWrapper from "./UnitsWrapper"

describe("UnitsWrapper", () => {
  it("renders with correct styling", () => {
    const { container } = render(<UnitsWrapper />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
