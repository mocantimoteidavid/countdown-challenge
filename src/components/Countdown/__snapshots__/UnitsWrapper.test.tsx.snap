// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnitsWrapper renders with correct styling 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: nowrap;
  -ms-flex-wrap: nowrap;
  flex-wrap: nowrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: left;
  -webkit-justify-content: left;
  -ms-flex-pack: left;
  justify-content: left;
}

.c0 > div:not(:last-child) {
  margin-right: 6px;
}

<div
  class="c0"
/>
`;
